js/src/behaviors/dismiss.ts

```diff
@@ -0,0 +1,42 @@
+export function initDismiss() {
+	window.addEventListener('click', ({ target }: Event) => {
+		const dismissButton = target instanceof Element && target.closest('[data-dismiss]');
+
+		if (!dismissButton) {
+			return;
+		}
+
+		const dismissTarget = dismissButton.closest('[data-dismissable]');
+
+		if (!dismissTarget) {
+			return;
+		}
+
+		dismissElement(dismissTarget);
+	});
+}
+
+async function dismissElement(dismissTarget: Element) {
+	const prefersReducedMotionReduceQuery = window.matchMedia('(prefers-reduced-motion: reduced)');
```

Comment:
Thinking that this can be remove in favor of a media query, see `animations.scss`

Reply:
The reason I did it here is to avoid 5s delay.

The alternative would be to use `animation-fade` for all cases, cause from my understanding `prefers-reduced-motion: reduced` doesn't necessarily mean no animation.

Reply:
Updated with Ben's suggestion, check the comment below